src/components/css-components/z-cover/index.stories.css

```diff
@@ -0,0 +1,10 @@
+.z-cover-story {
+  display: flex;
+  width: 220px;
+  height: 220px;
+  align-items: center;
+  justify-content: center;
+  padding: calc(var(--space-unit) * 2);
+  background-color: #f5f5f5;
```

Comment:
Vedo in giro colori assegnati tramite codice colore: si dovrebbero usare i token